youcube.lua

```diff
@@ -0,0 +1,18 @@
+-- https://github.com/Commandcracker/YouCube
+local package = {}
+package.name = "youcube"
+package.instdat = {}
+package.instdat.repo_owner = "Commandcracker"
+package.instdat.repo_name = "youcube"
```

Comment:
For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "YouCube"
```